### PR TITLE
updating roles and capabilities seeds

### LINKED ISSUE #2037: Seed roles, capabilities and api endpoints
- [x] Seed default roles
- [x] Seed default capabilities
- [x] Seed default api endpoints
- [ ] Seed default roles/capability mappings
- [x] Seed default capability/api mappings

The tables already exist and much of this seeding has already been done in the seeds.sql file, however, any gaps will need to be identified and resolved. I.e. new api endpoints have been created and need to be incorporated into the seeds.sql file.

### PR DESCRIPTION
**Pull request by mitchell852**
fixes #2037

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2211/
Test FAILed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2212/
Test FAILed.

File: traffic_ops/app/db/seeds.sql [hunk 1]
```diff
@@ -69,281 +69,502 @@ insert into status (name, description) values ('ADMIN_DOWN', 'Sever is administr
...
-insert into api_capability (http_method, route, capability) values ('PUT', '/api/*/types/*', 'type-write') ON CONFLICT (http_method, route, capability) DO NOTHING; -- 282
-insert into api_capability (http_method, route, capability) values ('DELETE', '/api/*/types/*', 'type-write') ON CONFLICT (http_method, route, capability) DO NOTHING; -- 283
-insert into api_capability (http_method, route, capability) values ('GET', '/api/*/users', 'user-read') ON CONFLICT (http_method, route, capability) DO NOTHING; -- 289
-insert into api_capability (http_method, route, capability) values ('GET', '/api/*/users/*', 'user-read') ON CONFLICT (http_method, route, capability) DO NOTHING; -- 290
-insert into api_capability (http_method, route, capability) values ('PUT', '/api/*/users/*', 'user-write') ON CONFLICT (http_method, route, capability) DO NOTHING; -- 292
-insert into api_capability (http_method, route, capability) values ('POST', '/api/*/users', 'user-write') ON CONFLICT (http_method, route, capability) DO NOTHING; -- 292
-insert into api_capabilit
```
Comment (moltzaum) [inline traffic_ops/app/db/seeds.sql]: `cachegroupparameters/*/*` is a DELETE request

Comment (moltzaum) [reply traffic_ops/app/db/seeds.sql]: Not the correct selection..

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2220/
Test FAILed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2221/
Test FAILed.

**Pull request merged**